ex03.exs

```diff
@@ -62,9 +62,14 @@ defmodule Ex03 do
   """
 
   def pmap(collection, process_count, function) do
-    « your code here »
+    size = collection |> Enum.count()
```

Comment:
the name `processes` confused me for the longest time, until I realized that it is actually nothing to do with processes: it's the chunk size :)

### -2